docs/dsp-tools-create.md

```diff
@@ -129,175 +122,70 @@ A simple example definition of the "project" object looks like this:
 }
 ```
 
-## Simple key/value pairs
+## "project" object in detail
 
-At that point we will go through all of this step by step and take a more in depth view on the individual fields of the
-"project" object. The first four fields of the "project" object are "key"/"value" pairs. Therefore, they are quite
-simple.
+In the following section all fields of the `project` object are explained in detail.
 
 ### Shortcode
 
 `"shortcode": "<4-hex-characters>"`
 
-It's a hexadecimal string in the range between "0000" and "FFFF" that's used to uniquely identify the project. The
-shortcode has to be provided by the DaSCH.
+Has to be unique and is represented by a 4 digit hexadecimal string. The shortcode has to be provided by the DaSCH.
 
 ### Shortname
 
 `"shortname": "<string>"`
 
-This is a short name (string) for the project. It's meant to be like a nickname. If the name of the project is e.g.
-"Albus Percival Wulfric Dumbledore", then the shortname for it could be "Albi". It should be in the form of a
-[xsd:NCNAME](https://www.w3.org/TR/xmlschema11-2/#NCName), that is a name without blanks and special characters like
-`:`, `;`, `&`, `%` etc., but `-` and `_` are allowed.
+Has to be unique. It should be in the form of a [xsd:NCNAME](https://www.w3.org/TR/xmlschema11-2/#NCName). This means a
```

Comment:
Make a complete sentence by adding the subject -> "The short name has  to be ....."